validation-test/stdlib/Index.swift.gyb

```diff
@@ -61,10 +61,11 @@ Index.test(
   "${Collection}/index(_:offsetBy:)/"
   + "avoidsSuccessorAndPredecessor/dispatch"
 ) {
-  for test in indexStepsFromTests.filter(
-    {$0.limit == nil && $0.distance >= 0}
+  let max = 10
+  for test in indexOffsetByTests.filter(
+    { $0.limit == nil && $0.distance >= 0 && $0.expectedOffset! < max }
```

Comment:
What if `expectedOffset` is `nil`? It is probably not in any of the cases that exist now, but just to make it future-proof, can you please check for `nil`ness as well?
```Swift
$0.expectedOffset.map {$0 < max} ?? false
```